src/university/universityParts/University.java

```diff
@@ -0,0 +1,85 @@
+package university.universityParts;
+
+import university.exceptions.nullListExceptions.NullFacultyListException;
+
+import java.text.DecimalFormat;
+
+/**
+ * University is a singletone class, as the program is written for a specific university.
```

Comment:
Karine jan your University is not singleton class as you have public constructor, remove these docs or make it singleton